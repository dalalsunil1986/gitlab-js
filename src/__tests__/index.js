const gitlab = require('../');

describe('request', () => {
  it('available methods', async () => {
    expect(gitlab).toMatchSnapshot();
  });
});
