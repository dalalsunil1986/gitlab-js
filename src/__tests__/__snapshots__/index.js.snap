// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`request available methods 1`] = `
Object {
  "mergeRequest": Object {
    "accept": [Function],
    "approve": [Function],
    "get": [Function],
    "list": [Function],
    "notes": [Function],
    "unaprove": [Function],
  },
  "project": Object {
    "all": [Function],
    "owned": [Function],
  },
}
`;
